DFC.ServiceTaxonomy.ContentPickerPreview/Views/ContentPickerField-Preview.Edit.cshtml

```diff
@@ -60,7 +60,7 @@
                 </vue-multiselect>
                 @if (!String.IsNullOrEmpty(settings.Hint))
                 {
-                    <span class="hint">@settings.Hint</span>
+                    <input ref="hint" type="hidden" value="@settings.Hint" />
```

Comment:
We shouldn't hide the hint. The occupation picker has a hint for example, and we might want to use it on other fields.

Reply:
Apologies, we did agree on using the hint. We can put the json after a token, that we use to cut off what we display in the front end.